test/models/user_test.rb

```diff
@@ -89,4 +89,10 @@ def setup
   test 'automatically assigns primary email on contact' do
     assert_equal @user.profile.contact.primary_email, @user.email
   end
+
+  test 'updating user email contact also on user' do
+    @user.profile.contact.update(primary_email: 'supervolunteer@example.com')
```

Comment:
`@user` is superadmin. I think you mentioned that the issue was with the volunteers.

Reply:
I noticed it first for a volunteer, but apparently it concerns all users.

Reply:
I am sorry, I hadn't noticed.